### PR TITLE
Convert two linear layout into one constraint

### PR DESCRIPTION
**Pull request by AndrewR8**
new project templates in Android Studio now use ConstraintLayout as default

the string name and the string itself are the same, and this is at least not beautiful, and it is not convenient for reading

### CODE REVIEW

**Comment by lukeswitz** [comment]
What bugs are you addressing here? The formatting of your code isn’t ready for a PR and the string change actually makes less sense. 

Please explain the purpose of these proposed changes.

**Comment by AndrewR8** [comment]
the string name and the string itself are the same, and this is at least not beautiful, and its not convinient for read

**Comment by AndrewR8** [comment]
new project templates in Android Studio now [use ConstraintLayout as default](https://classroom.udacity.com/courses/ud834/lessons/4034888704/concepts/5b798321-219a-4910-9be1-7fcac1e5b8dc)

**Comment by lukeswitz** [comment]
You updated the description, thank you. The string needs to be reworded nonetheless, I believe some of this is being lost in translation. 

There are other standards that need to be addressed before this, but I see what you’re trying to accomplish here. 

#275 needs a better solution imo; that’s all from me on this PR.

**Comment by lukeswitz** [comment]
The string is unused, changing it has nil imapact. UI feedback is adequate so far in users onboarding the decibel levels: “Make some noise” is an engaging phrase to let the user know it’s time to interact. Feedback latency is negligible compared to the motion preview and the dB levels are well explained.  

A full on UI overhaul is proposed in the issues, have a look at the clean work done in #218. 

You could help out #275 where the values are much more ambiguous than the audio sensor you’re working on here. Keep after it 👍

File: src/main/res/layout/activity_microphone_configure.xml [hunk 1]
```diff
@@ -1,5 +1,4 @@
-
-      <?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8"?>
 <android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (AndrewR8) [inline src/main/res/layout/activity_microphone_configure.xml]: @lukeswitz is it better?

Comment (lukeswitz) [reply src/main/res/layout/activity_microphone_configure.xml]: Negative, the changes here: 
screen shot 2018-04-21 at 11 29 20 am
indicate the opposite. I am not the place to learn proper PR's, please reference this in the meantime: http://blog.ploeh.dk/2015/01/15/10-tips-for-better-pull-requests/ 

Best,
Luke

Comment (AndrewR8) [reply src/main/res/layout/activity_microphone_configure.xml]: thank you, so much. sorry if i do something wrong. i'm just a beginner and there are a lot of new for me. thank you for patience

File: src/main/res/layout/activity_microphone_configure.xml [hunk 2]
```diff
@@ -20,100 +20,99 @@
...
-        android:textSize="20sp"
-        android:textStyle="bold"
-        app:layout_constraintBottom_toTopOf="@+id/userShouldDoHint"
-        app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintHorizontal_bias="0.5"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toBottomOf="@+id/toolbar"
-        app:layout_constraintVertical_weight="1" />
+        <TextView
+            android:id="@+id/userShouldDo"
```
Comment (lukeswitz) [inline src/main/res/layout/activity_microphone_configure.xml]: Remove indentation to actually correct the formatting (all structure has to match the original document). Use the spacing tools to highlight where you’re going wrong. It’s a pain I know. Good luck.

File: src/main/res/layout/activity_microphone_configure.xml (hunk 1 above)
Comment (lukeswitz) [reply src/main/res/layout/activity_microphone_configure.xml]: Nice work cleaning this up. I’d remove the string value change however as it is fine as it is (perhaps not localized in your language; if you want, add them to your PR and switch the English back).

**Comment by AndrewR8** [comment]
I'm really confused about what I need to do

**Comment by AndrewR8** [comment]
is everything okey with my commits? or i need  to change something else?

**Comment by n8fr8** [comment]
Sorry for the delay on reviewing this... taking a look now!

**Comment by n8fr8** [comment]
Regarding the string name, the naming was done through automated externalization using Android Studio. I agree it shouldn't be the key for the strong shouldn't be the english sentence with an underscore. 

That said, at this point changing keys for strings causes more problems than it solves, since we need to change them for all translations, etc, and push the updates strings and translations to Transifex.

**Pull request merged**